docs/eventing/samples/helloworld/helloworld-python/README.md

```diff
@@ -0,0 +1,338 @@
+A simple web app written in Python that you can use to test knative eventing. It shows how to consume a [CloudEvent](https://cloudevents.io/) in Knative eventing, and optionally how to respond back with another CloudEvent in the http response, by adding the Cloud Eventing headers outlined in the Cloud Events standard definition.
+
+We will deploy the app as a [Kubernetes Deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) along with a [Kubernetes Service](https://kubernetes.io/docs/concepts/services-networking/service/).
+However, you can also deploy the app as a [Knative Serving Service](../../../../serving/README.md).
+
+Follow the steps below to create the sample code and then deploy the app to your
+cluster. You can also download a working copy of the sample, by running the
+following commands:
+
+```shell
+# Clone the relevant branch version such as "release-0.13"
+git clone -b "{{< branch >}}" https://github.com/knative/docs knative-docs
+cd knative-docs/docs/eventing/samples/helloworld/helloworld-python
+```
+
+## Before you begin
+
```

Comment:
Add a line to say something like "To follow this example guide, you will need:"

Also, change "we'll use it"
I'd list the Docker Hub a/c as a separate point, e.g.
- A configured Docker Hub account to use as a container registry.